src/tasks/MotionForceTask.h

```diff
@@ -519,6 +527,36 @@ class MotionForceTask : public TemplateTask {
 		return _partial_task_projection.block<3, 3>(3, 3);
 	}
 
+	/**
+	 * @brief	Change the bounds for singularity cut-off
+	 * 
+	 * @param e_max 
+	 * @param e_min 
+	 */
+	void setSingularityBounds(const double& s_min, const double& s_max) {
+		_singularity_handler->setSingularityBounds(s_min, s_max);
+	}
+
+	MatrixXd getProjectedJacobian() {
+		return _projected_jacobian_ns;
+	}
+
+	VectorXd getUnitControlForces() {
+		return _unit_mass_force;
+	}
+
+	VectorXd getControlForces() {
+		return _Lambda_ns_modified * _task_range_ns.transpose() * _unit_mass_force;
+	}
+
+	MatrixXd getLambdaMatrix() {
+		return _Lambda_ns_modified * _task_range_ns.transpose();
+	}
+
+	const std::unique_ptr<SingularityHandler>& getConstSingularityModel() const {
```

Comment:
I am not sure we want this getter either, why would the user need acces to the internal singularity handler ?